bucket/confuser-ex.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.5.0",
+    "description": "An open-source protector for .NET applications",
+    "homepage": "https://github.com/mkaring/ConfuserEx",
```

Comment:
https://mkaring.github.io/ConfuserEx/

Reply:
"checkver": "github" won't work with this change

Reply:
What is this argument? You think that checkver could be only string? Like it would not have number of properties, which will support it

Reply:
Fixed